subprojects/docs/src/docs/userguide/authoring-builds/organizing_gradle_projects.adoc

```diff
@@ -95,8 +95,12 @@ Gradle tries to locate a `settings.gradle` (Groovy DSL) or a `settings.gradle.kt
 For that purpose, the runtime walks the hierarchy of the directory tree up to the root directory.
 The algorithm stops searching as soon as it finds the settings file.
 
+A multi-project build must have a `settings.gradle` file in the root project of the multi-project hierarchy.
```

Comment:
Similar idea here for `settings.gradle` as above.

Reply:
```suggestion
A multi-project build must have a `settings.gradle(.kts)` file in the root project of the multi-project hierarchy.
```